kubectl-plugin/pkg/cmd/create/create_cluster.go

```diff
@@ -0,0 +1,223 @@
+package create
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ray-project/kuberay/kubectl-plugin/pkg/util/generation"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+type CreateClusterOptions struct {
+	configFlags    *genericclioptions.ConfigFlags
+	ioStreams      *genericclioptions.IOStreams
+	clusterName    string
+	fileName       string
+	rayVersion     string
+	image          string
+	headCPU        string
+	headMemory     string
+	workerGrpName  string
+	workerCPU      string
+	workerMemory   string
+	workerReplicas int32
+}
+
+var (
+	createClusterLong = templates.LongDesc(`
+		Creates Ray Cluster from inputed file or generate one for user.
+	`)
+
+	createClusterExample = templates.Examples(`
+		# Creates Ray Cluster from inputted Yaml
+		kubectl ray create cluster -f raycluster-stample.yaml
```

Comment:
Do we need to support this? This isn't much different from `kubectl apply -f raycluster-sample.yaml` right?

Reply:
Yes, this would be the same as `kubectl apply`.